wvc-ourwork-nextjs/icons/QuestionMark.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+export const QuestionMark = () => {
+  return (
+    <svg
+      width='16'
+      height='16'
+      viewBox='0 0 16 16'
+      fill='none'
+      xmlns='http://www.w3.org/2000/svg'
+    >
+      <path
+        fill-rule='evenodd'
+        clip-rule='evenodd'
```

Comment:
All good here! One tiny thing to remove "Invalid DOM property" is to change these to 
`fillRule='evenodd'`
 `clipRule='evenodd'`
 
 Otherwise! 🚢

Reply:
Goot catch!